ui/apps/platform/src/Containers/VulnMgmt/widgets/TopRiskyEntitiesByVulnerabilities.js

```diff
@@ -14,19 +14,24 @@ import HoverHintListItem from 'Components/visuals/HoverHintListItem';
 import TextSelect from 'Components/TextSelect';
 import entityTypes from 'constants/entityTypes';
 import entityLabels from 'messages/entity';
-import { severityLabels } from 'messages/common';
-import {
-    severityColorMap,
-    severityTextColorMap,
-    severityColorLegend,
-} from 'constants/severityColors';
+import { policySeverityColorMap } from 'constants/visuals/colors';
+import { severityLabels as policySeverityLabels } from 'messages/common';
 import { checkForPermissionErrorMessage } from 'utils/permissionUtils';
 import { getSeverityByCvss } from 'utils/vulnerabilityUtils';
 import { entitySortFieldsMap, cveSortFields } from 'constants/sortFields';
 import { WIDGET_PAGINATION_START_OFFSET } from 'constants/workflowPages.constants';
 import { entityPriorityField } from 'Containers/VulnMgmt/VulnMgmt.constants';
 import useFeatureFlags from 'hooks/useFeatureFlags';
 
+// Beware, policy instead of vulnerability severities because of getSeverityByCvss function!
+
+const policySeverities = ['LOW_SEVERITY', 'MEDIUM_SEVERITY', 'HIGH_SEVERITY', 'CRITICAL_SEVERITY'];
```

Comment:
Nit: what about importing this from `policy.proto.ts`? (Ditto on the above file)

Reply:
Yes, indeed. I forgot that the opposite order from Critical to Low in legend of `PolicyViolationsBySeverity` is independent of the order of the severity in the arrays.